presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/QueryBuilder.java

```diff
@@ -76,9 +78,12 @@ public Object getValue()
         }
     }
 
-    public QueryBuilder(String quote)
+    public static final Function<JdbcColumnHandle, String> DEFAULT_BINDING_PLACEHOLDER = column -> "?";
+
+    public QueryBuilder(String quote, Function<JdbcColumnHandle, String> bindingPlaceholder)
```

Comment:
Please provide second constructor:
```
QueryBuilder(String quote)
{
   this(quoute, DEFAULT_BINDING_PLACEHOLDER);
}
```

Then please use it in places where you are passing `DEFAULT_BINDING_PLACEHOLDER` directly.